Library/Formula/libjit.rb

```diff
@@ -0,0 +1,19 @@
+class Libjit < Formula
+  homepage "https://www.gnu.org/software/libjit/"
+  url "git://git.savannah.gnu.org/libjit.git"
+  version "0.1.3"
```

Comment:
It looks like 0.1.3 hasn't been released yet - we can't assign arbitrary versions to this.